src/GitHub.Extensions/UriExtensions.cs

```diff
@@ -25,6 +25,11 @@ public static bool IsHypertextTransferProtocol(this Uri uri)
             return uri.Scheme == "http" || uri.Scheme == "https";
         }
 
+        public static bool IsGithubUri(this Uri uri)
+        {
+            return uri.OriginalString.Contains("github.com");
```

Comment:
`String.Contains` is case and culture insensitive, should I should change this` .ToLower()`?
Also I wonder if using `String.StartsWith(https://www.github.com)` is a better alternative?